src/Acf.php

```diff
@@ -185,4 +199,23 @@ private static function apply_default_transform( $field_obj ) {
 	private static function get_transform_class_name( $field_type ) {
 		return '\\' . __NAMESPACE__ . '\\Acf\\Transforms\\' . str_replace( '_', '', ucwords( $field_type, '_' ) );
 	}
+
+	/**
+	 * Get an ACF group
+	 *
+	 * @param $group_id
+	 * @return bool
+	 */
+	private static function get_group( $group_id ) {
+		// @codingStandardsIgnoreStart
+		// Ignore use WP_Query rule because we don't know if this will be a sub-query and hence create complications.
```

Comment:
We can restart the post data. But I was taking a quick look at this and I'm pretty sure this can be simplified with this lines: 

```php
$group = get_page_by_path( $group_id', OBJECT, 'acf-field-group');
if ( $group ) {
    $title = $group->post_title;
}
```